pyangbind/plugin/pybind.py

```diff
@@ -1611,7 +1618,8 @@ def get_element(ctx, fd, element, module, parent, path,
         "choice": choice,
         "register_paths": register_paths,
         "namespace": namespace,
-        "defining_module": defining_module
+        "defining_module": defining_module,
+        "original_module": element.i_orig_module.arg,
```

Comment:
Can `i_orig_module` ever be `None`, or can `element` not have `i_orig_module` as an attribute? 

The case above seems to show that it can be, but this one just references it directly.